### PR TITLE
add bespoke res.render sink for express tb

### PR DESCRIPTION
**Pull request by Avuxo**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -64,6 +64,7 @@ app.use('/typecheck', require('./vulnerabilities/typecheck'));
 app.use('/mongoose', require('./vulnerabilities/mongoose'));
 app.use('/express-session', require('./vulnerabilities/express-session'));
 app.use('/ddb', require('./vulnerabilities/dynamodb'));
```
Comment (bizob2828) [inline index.js:66]: We built the test bench utils and such to normalize a lot across frameworks. I get this is a 1 off but this sink still applies to xss. Can you follow the `/<rule>/<sink>/unsafe|safe` model? Feels like this work is also out of scope and should be its own ticket to do right and not just rushed to show this sink works functionally

**Review by bizob2828 — CHANGES_REQUESTED** [review]

File: vulnerabilities/render/index.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+'use strict';
+const express = require('express');
+const api = express.Router();
+
+api.get('/', (req, res) => {
+  res.render(`${__dirname}/views/index`);
```
Comment (pmcclory-contrast) [inline vulnerabilities/render/index.js:6]: a nitpick but can we just render the same template in both cases, and just not pass the options?

File: index.js (hunk 1 above)
Comment (Avuxo) [reply index.js:66]: If that's team consensus, I wouldn't mind declining this PR and deferring to a ticket.

Comment (pmcclory-contrast) [reply index.js:66]: works for me.
